backend/src/packages/chaiNNer_standard/utility/directories/into_directory.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+from nodes.properties.inputs import BoolInput, DirectoryInput, TextInput
+from nodes.properties.outputs import DirectoryOutput
+
+from .. import value_group
+
+
+@value_group.register(
+    schema_id="chainner:utility:into_directory",
+    name="Into Directory",
+    description="Goes forward into a directory.",
+    icon="BsFolder",
+    inputs=[
+        DirectoryInput(
+            "Directory", must_exist=False, label_style="hidden", has_handle=True
+        ),
+        TextInput("Folder", has_handle=True),
+        BoolInput("Create if not exists", default=True),
+    ],
+    outputs=[
+        DirectoryOutput(
+            "Directory",
+            output_type="Directory { path: combinePath(Input0.path, Input1) }",
+        ),
+    ],
+)
+def into_directory_node(directory: Path, folder: str, create_path: bool) -> Path:
+    resolved = (directory / folder).resolve()
+    if create_path and not resolved.exists():
+        resolved.mkdir(parents=True)
+    elif not resolved.exists():
+        raise FileNotFoundError(f"Directory does not exist: {resolved}")
```

Comment:
So the directory must always exist?

Reply:
Yes. Do you not think so? I don't think we should be passing in invalid directories into nodes willy nilly

Reply:
Yeah, I don't like it.

1. This means that the correctness of a chain now depends on the file system. If user A has folder `foo` but user B doesn't, it's not gonna work for user B unless they check that box. Depending on the file system is unavoidable for node like Load Image (singular), but it can be avoided here.
2. This leads to errors being ignored. Nodes like Load Images don't allow non-existent directories as input for obvious reasons. So if you always just create those directories, then Load Images can't error for incorrect chains.
3. We already allow passing non-existent directories. Save Image and the Directory node support it.
4. Side effects. You turned this node into a side effect node.